setup.py

```diff
@@ -31,11 +31,14 @@
         ],
         packages=['kama'],
         install_requires=[
```

Comment:
Seems awkward to have both install_requires in setup.py and requirements.txt. One or the other, not both.

Reply:
You can add tests_require to setup.py